bundles/org.openhab.binding.freeboxos/README.md

```diff
@@ -0,0 +1,124 @@
+
+| Parameter Label  | Parameter ID    | Description                                                                | Required | Default |
+|------------------|-----------------|----------------------------------------------------------------------------|----------|---------|
+| MAC Address      | macAddress      | The MAC address of the network host .                                      | Yes      |         |
+| Refresh Interval | refreshInterval | The refresh interval in seconds which is used to poll for phone state.     | No       | 30      |
+
+### Repeater and Vm thing 
+
+The *repeater* thing is a specialized case of a *wifihost*. The *vm* derives from *host*. They share the same configuration definition : 
+
+| Parameter Label  | Parameter ID    | Description                                                      | Required | Default |
+|------------------|-----------------|------------------------------------------------------------------|----------|---------|
+| MAC Address      | macAddress      | The MAC address of the player device.                            | No       |         |
+| ID               | id              | Id of the repeater within Freebox Api                            | Yes      | 1       |
+| Refresh Interval | refreshInterval | The refresh interval in seconds which is used to poll the player | Yes      | 30      |
+
+## Authentication
+
+You will have to authorize openHAB to connect to your Freebox. Here is the process described :
+
+**Step 1** At binding startup, if no token is recorded in the Freebox Server (bridge) configuration, the binding will run a pairing request
+
+**Step 2** Run to your Freebox and approve the pairing request for openHAB Freebox Binding that is displayed on the Freebox screen
+
+**Step 3** the application token is automatically recorded in the Freebox Server (bridge) configuration
+
+**Step 4** you can use the console command `freebox <bridgeUID> apptoken` to display the application token and use it later to set up your thing in a configuration file
+
+**Optionally** you can log in your Freebox admin console to allocate needed rights to openHAB
+
+Once initialized, the thing will generate all available channels.
+
+## Actions
+
+| Thing Type  | Action Name    | Description                                           | 
+|-------------|----------------|-------------------------------------------------------|
+| host        | wol            | Sends a wake on lan packet to the lan connected host. |
+| player      | reboot         | Reboots the player device.                            |
+| player      | sendKey        | Send a key (remote emulation) to the player           |
+| player      | sendLongKey    | Sends the key emulating a longpress on the button     |
+| player      | sendKeyRepeat  | Sends the key multiple times                          |
+| server      | reboot         | Reboots the Freebox Server.                           |
+

```

Comment:
I think you should add a last section listing all features that are no more present in this new version of the binding, compared to the current binding.

Reply:
There should not be missing channels between the two versions, maybe only new ways to access the same info.

Reply:
Ok for the channels, I will compare later. So maybe just add a sentence in this new section mentioning that all channels from the old binding are still available but sometimes with a different name and on a different thing.

In this case, just focus on missing features. In the current binding, I distinguished between network device and network interface to monitor reachability. You did it, I believe, only on network devices. So add something like "Monitoring of network interfaces reachability is no more available while monitoring of network devices reachability is still available."

Reply:
A missing feature to be mentioned: no more possible to monitor the reachability of a particular IP address. Only the reachability of a device defined by its MAC is now available.